rdr_service/dao/bq_pdr_participant_summary_dao.py

```diff
@@ -214,8 +213,7 @@ def _calculate_ubr(self, ps_bqr, p_id):
                         break
 
             if consent_date:
-                age = int((dateutil.parser.parse(consent_date.isoformat()) -
-                                dateutil.parser.parse(ps_bqr.date_of_birth)).days / 365)
+                age = int((consent_date - ps_bqr.date_of_birth).days / 365)
```

Comment:
This doesn't account for leap years.

Reply:
Leap years can be a future enhancement if needed.

Reply:
The final calculation from this is just to see if the participant's age is between 18 and 65.

Reply:
Not sure if this was the only place you used dateutil.parser and if that is now an unused import.  Pylint may not like that on push.

Reply:
How accurate is it expected that to be? This will be incorrect for people up to 4 days before they turn 18 and 16 days before their 65th birthday.

What is the number used for? Is it worth the risk if it would mark someone that gave consent before they were 18?

Reply:
Let me qualify this, its calculating the age of the participant at the time they consented to the program.  So leap year is not a factor since both values are fixed and don't change over time.  Its not calculating the participant's current age.